vignettes/rd-formatting.Rmd

```diff
@@ -371,18 +361,53 @@ Plots will create `.png` files in the `man/figures` directory. The file names ar
 #' ```
 ```
 
-Note that the generated `.png` files will be added to the package, and they can considerably increase the size of the package.
+## Gotchas
 
-Note that code block support is currently experimental, and somewhat limited.
-Some of the known limitations:
+This feature is powered by the knitr package.
 
-  * Because the code blocks are evaluated individually, they cannot refer to each other.
+This causes some limitations and differences to keep in mind:
+
+  * Inline and fenced code blocks from the same _roxygen block_ share the same evaluation environment. 
```

Comment:
This doesn't seem like a gotcha to me. Maybe "Details" would be a better heading?